azkaban-common/src/test/java/azkaban/project/AzkabanProjectLoaderTest.java

```diff
@@ -282,10 +290,12 @@ public void uploadProjectWithYamlFilesFAT() throws Exception {
         .thenReturn(flowVersion);
 
     checkValidationReport(this.azkabanProjectLoader
-        .uploadProject(this.project, projectZipFile, "zip", uploader, null));
+        .uploadProject(this.project, projectZipFile, "zip", uploader, null,
+            "111.111.111.111"));
 
     verify(this.projectStorageManager)
-        .uploadProject(this.project, this.VERSION + 1, projectZipFile, null, uploader);
+        .uploadProject(this.project, this.VERSION + 1, projectZipFile,
+            null, uploader, "111.111.111.111");
```

Comment:
Would you like to use the longest IPv6 address in some of the tests? Also maybe make it a (couple of) const?

Reply:
Will do both. Thanks